modules/core/src/main/java/org/locationtech/jts/io/gml2/GeometryStrategies.java

```diff
@@ -336,6 +337,11 @@ else if(arg.attrs.getIndex(GMLConstants.GML_NAMESPACE,"ts")>=0)
 				// now to start parse
 				String t = arg.text.toString();
 				t = t.replaceAll("\\s"," ");
+				/**
+				 * Remove spaces after commas, for when they are used as separators (default).
+				 * This prevents coordinates being split by the tuple separator
+				 */
+				t = t.replaceAll(",\\s+", ",");
```

Comment:
I think we should also handle spaces before comma as well?
```
<gml:Point gml:id="p21" srsName="http://www.opengis.net/def/crs/EPSG/0/4326">
    <gml:coordinates>45.67     ,88.56</gml:coordinates>
 </gml:Point>
```

Reply:
yes, probably should

Reply:
Thanks!

Reply:
Done!